src/schemas/tag.ts

```diff
@@ -18,7 +19,7 @@ export const typeDefs = gql`
   "A Tag is a way to group similar types of categories together"
   type Tag {
     "The unique identifer for the Object"
-    id: Int!
+    id: Int
```

Comment:
This "id" is auto-incremented in the database, so it's not required by the user.

Reply:
Yes good catch.